src/core_codemods/fix_task_instantiation.py

```diff
@@ -0,0 +1,96 @@
+import libcst as cst
+from typing import Optional
+from core_codemods.api import Metadata, ReviewGuidance, SimpleCodemod, Reference
+from codemodder.codemods.utils_mixin import NameAndAncestorResolutionMixin
+from codemodder.codemods.utils import BaseType, infer_expression_type
+
+
+class FixTaskInstantiation(SimpleCodemod, NameAndAncestorResolutionMixin):
+    metadata = Metadata(
+        name="fix-task-instantiation",
```

Comment:
```suggestion
        name="fix-async-task-instantiation",
```
Probably should be reflected in the name of the class/file as well 🙏